packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-});
-
-QUnit.test('input maxlength is updated when setting maxlength property of view', function() {
-  equal(view.$('input').attr('maxlength'), '30', 'renders text field with maxlength');
-  run(null, set, controller, 'max', 40);
-  equal(view.$('input').attr('maxlength'), '40', 'updates text field after maxlength changes');
-});
-
-QUnit.test('input size is updated when setting size property of view', function() {
-  equal(view.$('input').attr('size'), '30', 'renders text field with size');
-  run(null, set, controller, 'size', 40);
-  equal(view.$('input').attr('size'), '40', 'updates text field after size changes');
-});
-
-QUnit.test('input tabindex is updated when setting tabindex property of view', function() {
-  equal(view.$('input').attr('tabindex'), '5', 'renders text field with the tabindex');
-  run(null, set, controller, 'tab', 3);
-  equal(view.$('input').attr('tabindex'), '3', 'updates text field after tabindex changes');
-});
-
-QUnit.test('cursor position is not lost when updating content', function() {
-  equal(view.$('input').val(), 'hello', 'precondition - renders text field with value');
-
-  var $input = view.$('input');
-  var input = $input[0];
-
-  // set the cursor position to 3 (no selection)
-  run(function() {
-    input.value = 'derp';
-    view.childViews[0]._elementValueDidChange();
-    input.selectionStart = 3;
-    input.selectionEnd = 3;
-  });
-  run(null, set, controller, 'val', 'derp');
-
-  equal(view.$('input').val(), 'derp', 'updates text field after value changes');
-
-  equal(input.selectionStart, 3, 'cursor position was not lost');
-  equal(input.selectionEnd, 3, 'cursor position was not lost');
-});
-
-QUnit.test('input can be updated multiple times', function() {
-  equal(view.$('input').val(), 'hello', 'precondition - renders text field with value');
-
-  var $input = view.$('input');
-  var input = $input[0];
-
-  run(null, set, controller, 'val', '');
-  equal(view.$('input').val(), '', 'updates first time');
-
-  // Simulates setting the input to the same value as it already is which won't cause a rerender
-  run(function() {
-    input.value = 'derp';
-  });
-  run(null, set, controller, 'val', 'derp');
-  equal(view.$('input').val(), 'derp', 'updates second time');
-
-  run(null, set, controller, 'val', '');
-  equal(view.$('input').val(), '', 'updates third time');
-});
-
-
-QUnit.module('{{input type=\'text\'}} - static values', {
-  setup() {
-    commonSetup();
-
-    controller = {};
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller: controller,
-      template: compile('{{input type="text" disabled=true value="hello" placeholder="Enter some text" name="some-name" maxlength=30 size=30 tabindex=5}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
-  equal(view.$('input').length, 1, 'A single text field was inserted');
-});
```

Comment:
moved to `@htmlbars Helpers test: {{input}}` : [`@test a single text field is inserted into the DOM`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R71) (1)